platforms/ios/HelloCordova/Plugins/WizAssetsPlugin/WizAssetsPlugin.m

```diff
@@ -63,15 +63,25 @@ - (void)backgroundDownload:(NSDictionary *)args {
             bool urlUnauthorized = [dataContent isEqualToString:@"Unauthorized"];
             [dataContent release];
 
+            NSError *error = nil;
```

Comment:
I would rather rename it to differentiate it from the one declared at l.54 or not declare it at all. Not a blocker however.